### PR TITLE
Remove some unused code

### PR DESCRIPTION
**Pull request by pzygielo**

### CODE REVIEW

File: api/src/main/java/jakarta/el/StaticFieldELResolver.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public Object getValue(ELContext context, Object base, Object property) {
                 Field field = klass.getField(fieldName);
 
                 int mod = field.getModifiers();
-                if (Modifier.isPublic(mod) && Modifier.isStatic(mod)) {
+                if (isPublic(mod) && isStatic(mod)) {
                     return field.get(null);
```
Comment (markt-asf) [inline api/src/main/java/jakarta/el/StaticFieldELResolver.java:84]: I'd prefer to see the static imports removed but I guess that falls under personal style preference. Consistent is certainly better that what we have now.

File: impl/src/main/java/com/sun/el/parser/ELParserTokenManager.java [hunk 2]
```diff
@@ -16,8 +16,6 @@
  
 /* Generated By:JJTree&JavaCC: Do not edit this line. ELParserTokenManager.java */
 package com.sun.el.parser;
-import java.io.StringReader;
-import jakarta.el.ELException;
```
Comment (markt-asf) [inline impl/src/main/java/com/sun/el/parser/ELParserTokenManager.java:19]: As per my previous comment (that didn't appear where/how I expected it to) this is a generated class. There isn't much point cleaning it up.

**Comment by pzygielo** [comment]
@markt-asf thanks for checking, rebased w/o commit for `ELParserTokenManager.java`.

File: api/src/main/java/jakarta/el/StaticFieldELResolver.java (hunk 1 above)
Comment (pzygielo) [reply api/src/main/java/jakarta/el/StaticFieldELResolver.java:84]: Methods already are imported and used.

**Comment by markt-asf** [comment]
Thanks for the PR and for responding promptly to my review comments.

**Pull request merged**